examples/gmail_oauth2.rs

```diff
@@ -20,6 +21,7 @@ impl imap::Authenticator for GmailOAuth2 {
     }
 }
 
+#[cfg(feature = "ssl")]
```

Comment:
It's probably better here to list the example in `Cargo.toml` with `required-features = ["ssl"]` (assuming that works).

Reply:
I didn't know about `required-features`. Seems like that's a much better solution, thanks.